pkg/kubectl/cmd/get.go

```diff
@@ -182,13 +183,22 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 		return cmdutil.UsageError(cmd, usageString)
 	}
 
-	// always show resources when getting by name or filename
+	// determine if args contains "all"
+	for _, a := range args {
+		if a == "all" {
```

Comment:
> Maybe we should check there is only 1 argument, and it is all?

Sounds good to me, I can go ahead and open a PR to patch this in the `MultipleTypesRequested` helper